meilisearch/src/search.rs

```diff
@@ -523,6 +553,20 @@ fn insert_geo_distance(sorts: &[String], document: &mut Document) {
     }
 }
 
+fn insert_semantic_score(query: &[f32], vectors: Value, document: &mut Document) {
+    let vectors =
+        match serde_json::from_value(vectors).map(VectorOrArrayOfVectors::into_array_of_vectors) {
+            Ok(vectors) => vectors,
+            Err(_) => return,
```

Comment:
It is unclear to me what error conditions we might be ignoring here. Are they always benign?

Reply:
I just returned an Internal error when this happened. Thank you :)